python/ray/serve/task_processor.py

```diff
@@ -35,6 +38,43 @@ class AsyncCapability(Enum):
     HEALTH_CHECK = auto()  # Ability to perform health checks asynchronously
 
 
+def _make_json_serializable(obj: Any) -> Any:
```

Comment:
or something similar

```suggestion
def json_dump(obj: Any) -> Any:
```